challenge.md

```diff
@@ -32,19 +32,19 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
 <!-- Answer Starts Here -->
-
+Absolute path -- starts at "Users" so you know the computer will recognize it wherever you are.
```

Comment:
You're correct that it's an absolute path - this path is totally independent of out position. However, your reasoning re: starting at "Users" isn't quite right - the real key is that the path starts in the root, a totally unambiguous location as far as the filesystem is concerned.